src/tests/FunctionalTests/iOS/Simulator/InvariantCultureModeOnly/Program.cs

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Runtime.InteropServices;
+
+public static class Program
+{
+    [DllImport("__Internal")]
+    public static extern void mono_ios_set_summary (string value);
+    
+    public static async Task<int> Main(string[] args)
+    {
+        mono_ios_set_summary($"Starting functional test");
+        int result = PlatformDetection.IsInvariantGlobalization ? 42 : 1;
```

Comment:
It'd be better to call api which behaves differently with invariant mode and test the result

Reply:
I guess something similar to https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.Extensions.Primitives/tests/StringSegmentTest.cs#L1275-L1276 should work here

Reply:
I think even simpler check like `new CultureInfo("es-ES", false) == Invariant` should do it

Reply:
Updated the test